src/Dfe.PlanTech.Application/Users/Commands/RecordUserSigninCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Dfe.PlanTech.Application.Persistence.Interfaces;
+using Dfe.PlanTech.Application.Users.Interfaces;
+using Dfe.PlanTech.Application.Users.Queries;
+using Dfe.PlanTech.Domain.SignIn.Models;
+using Dfe.PlanTech.Domain.Users.Models;
+
+namespace Dfe.PlanTech.Application.Users.Commands;
+
+public class RecordUserSignInCommand : IRecordUserSignInCommand
+{
+    private readonly IUsersDbContext _db;
+
+    public RecordUserSignInCommand(IUsersDbContext db)
+    {
+        _db = db;
+    }
+
+    public async Task<int> RecordSignIn(RecordUserSignInDto recordUserSignInDto)
+    {
```

Comment:
I feel we could separate out the concerns here for single responsibility - RecordSignIn is getting users, possibly creating users as well as AddSignInDetails

Reply:
Replaced so the creation of the user is handled by the CreateUserCommand so all creation is done there.